rustworkx/visualization/graphviz.py

```diff
@@ -185,20 +184,18 @@ def node_attr(node):
         prog = method
 
     if not filename:
-        with tempfile.TemporaryDirectory() as tmpdirname:
-            filename = f"graphviz_draw_{str(uuid.uuid4())}.{output_format}"
-            tmp_path = os.path.join(tmpdirname, filename)
-            subprocess.run(
-                [prog, "-T", output_format, "-o", tmp_path],
-                input=dot_str,
-                check=True,
-                encoding="utf8",
-                text=True,
-            )
-            with Image.open(tmp_path) as temp_image:
-                image = temp_image.copy()
-            os.remove(tmp_path)
-            return image
+        dot_result = subprocess.run(
+            [prog, "-T", output_format],
+            input=dot_str.encode("utf-8"),
+            capture_output=True,
+            encoding=None,
+            check=True,
+            text=False,
+        )
+        dot_bytes_image = io.BytesIO(dot_result.stdout)
+        with Image.open(dot_bytes_image) as temp_image:
+            image = temp_image.copy()
```

Comment:
Do we still need the copy here? IIRC I added this to avoid a different windows issue where we couldn't delete the tempfile the image was stored in on windows if something had an open handle to it. So we copied it and returned the PIL object wrapping the copy to close all usage of the tempfile and avoid the windows failure so we could delete the retworkx/rustworkx created tempfile and let pillow do the file management for it's objects.